### PR TITLE
[Pluggable Device] Add DEFAULT_DEVICE registrations for some ops.

### PR DESCRIPTION
**Pull request by kulinseth**
Adding some more registrations for commonly used ops to DEFAULT_DEVICE using CPU implementation.  
cc @penpornk

### CODE REVIEW

**Review by penpornk — CHANGES_REQUESTED** [review]
Ask discussed offline, all registrations in this PR had caused internal test failures and were reverted before, and the issues haven't been fixed yet. Please guard all device default registrations in this PR with macos ifdefs for now.

**Comment by kulinseth** [comment]
> Ask discussed offline, all registrations in this PR had caused internal test failures and were reverted before, and the issues haven't been fixed yet. Please guard all device default registrations in this PR with macos ifdefs for now.

Thanks @penpornk , added them to be specific to macos.

**Review by cantonios — APPROVED** [review]

**Review by penpornk — CHANGES_REQUESTED** [review]

**Comment by penpornk** [comment]
For the record: I had made the requested changes internally before merging the PR.

**Comment by kulinseth** [comment]
> For the record: I had made the requested changes internally before merging the PR.

Thanks Penporn. Much appreciated.

File: tensorflow/core/kernels/pack_op.cc [hunk 1]
```diff
@@ -163,4 +163,15 @@ REGISTER_KERNEL_BUILDER(Name("Pack")
 
 #endif  // GOOGLE_CUDA || TENSORFLOW_USE_ROCM
 
+#if defined(PLUGGABLE_DEVICE_SUPPORTED_MACOS)
```
[the remark this answers is not in the document]
Comment (kulinseth) [reply tensorflow/core/kernels/pack_op.cc:166]: Yes, i was thinking of moving it to `op_kernel.h`. But I wasn't sure as there were other Macros replicated across .cc file.

**Pull request merged**